ui/app/components/select-dropdown.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@ember/component';
+
+/**
+ * @module SelectDropdown
+ * SelectDropdown components are used to render a dropdown.
+ *
+ * @example
+ * ```js
+ * <SelectDropdown dropdownLabel='Date Range' @options={{options}} @onChange={{onChange}}/>
```

Comment:
Was there a reason for not using `Select` for the name of the component?

Reply:
yeah, i wanted to name it `Select` but i received the 'must include hyphen in a component name' error when generating the component.  it looks like the issue was solved by https://github.com/emberjs/ember.js/issues/17909 and included in ember-source 3.7 but we're only at 3.4. i can go back and rename the component though! 

meanwhile, is there any reason why we haven't upgraded to ember-source 3.7?

Reply:
Ah yeah I forgot about that. Re: upgrading - it's just timing really - we should upgrade to 3.8 when we get a chance (that's the current LTS).